algorithms/isomorphism_problem.py

```diff
@@ -0,0 +1,39 @@
+# import
+
+
+"""
+With these methods, the graph isomorphism problem can be solved.
+"""
+
+
+def amount_of_automorphisms(G):
```

Comment:
Volgens mij gebruik je deze methode nu niet in de test. Dan zou ik hem ook weglaten omdat het eigenlijk alleen `amount_of_isomorphisms` aanroept die je wel test in de test.